README.md

```diff
@@ -1,24 +1,167 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-* Ruby version
 
-* System dependencies
+# 📖 Budget app project <a name="about-project"></a>
+<br>
+This Ruby on Rails project is about a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.<br>
+<br>
 
-* Configuration
+## 🛠 Built With <a name="built-with"></a><br>
 
-* Database creation
+### Tech Stack
 
-* Database initialization
+This project is built using the following technologies:
 
-* How to run the test suite
+<details>
+  <summary>Programming languages</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on rails</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<br>
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+<br>
+
+#### The app allows the user to:
+
+- **Register and log in, so that the data is private to them.**
+- **Introduce new transactions associated with a category.**
+- **See the money spent on each category**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo & Video presentation<a name="live-demo"></a> <a name='video'></a>
+<br>
+
+- [Check the live demo, mobile version](https://github.com/rbd3/Budget-App/tree/deploy)
```

Comment:
- kindly, deploy the application using Render, Netlify or GitHub Pages to fit the requirment :+1: